src/app/components/playlist/playlist.component.ts

```diff
@@ -199,7 +199,7 @@ export class PlaylistComponent implements OnInit {
   }
 
   private increaseOffset() {
-    this.offset += this.limit - 1
+    this.offset += this.limit - 2
```

Comment:
Why are you changing this?

Reply:
This will break our query for more tracks. For instance if we have a limit of 10 and an offset of 8 we will have duplicate tracks in our list. What this method is supposed to do is adjust for index and then increment the offset by the index of the limit.

Reply:
After looking into it, we actually don't need to adjust for index. That means we can just do something like this:
```typescript
  private increaseOffset() {
    this.offset += this.limit
  }
```